plugins/accuracy.cc

```diff
@@ -286,30 +290,38 @@ runCategorical(AccuracyConfig & runAccuracyConf,
                     }
 
                     if (accuracyOverN > 1) {
-                        bestlabelsCandidates.emplace_back(v, 
-                            CellValue(jsonDecodeStr<CellValue>(columnName.toSimpleName())));
+                        bestlabelsCandidates[labelCount] = std::pair<double, CellValue>(v, jsonDecodeStr<CellValue>(columnName.toSimpleName()));
+                        ++labelCount;
                     }
 
                     return true;
                 };
             scoreLabelWeight[0].forEachAtom(onAtom);
 
             if (bestlabelsCandidates.size() > accuracyOverN) {
-                std::sort(bestlabelsCandidates.begin(), 
-                          bestlabelsCandidates.end(), 
+                std::partial_sort(bestlabelsCandidates.data(), 
+                          bestlabelsCandidates.data() + accuracyOverN,
+                          bestlabelsCandidates.data() + bestlabelsCandidates.size(), 
                           std::greater<std::pair<double, CellValue>>());
```

Comment:
Danger danger: if there are NaN values, this std::sort will segfault as x < y and y < x simultaneously.  You need to use a safe wrapper that handles NaN.